redirects/main.tf

```diff
@@ -209,6 +209,14 @@ resource "fastly_service_v1" "redirects2" {
     name = "vote.dosomething.org"
   }
 
+  domain {
+    name = "importer.dosomething.org"
+  }
+
+  domain {
+    name = "importer-qa.dosomething.org"
+  }
```

Comment:
This attaches these domains to this Fastly property (so Fastly knows what do with incoming traffic).